packages/smart-wallet/src/smartWallet.js

```diff
@@ -319,6 +340,7 @@ export const prepareSmartWallet = (baggage, shared) => {
             offerToPublicSubscriberPaths: [
               ...offerToPublicSubscriberPaths.entries(),
             ],
+            liveOffers: [...liveOffers.entries()],
```

Comment:
The name suggests that we're publishing offers here... offers in some ways include payments... which suggests anybody could get access to the payments via the board.

But I suppose we're only publishing `offerStatus`s here, which we're clearly already doing.

In the future, please be explicit about having thought through security considerations, following the PR template.

Reply:
Ah... and we _do_ iterate over all of these, without regard to input message size... which, strictly speaking, goes against the [limit work by message size](https://github.com/Agoric/agoric-sdk/wiki/Limit-work-by-message-size) rule.

But as discussed in #6184, there are somewhat conflicting requirements here, so I guess we can go with this.